src/utils/utils.ts

```diff
@@ -1,3 +1,22 @@
 export function clone(obj) {
 	return JSON.parse(JSON.stringify(obj));
 }
+
+function matchesAttr(el, attr, val) {
+	const styles = window.getComputedStyle(el);
+	return val.includes(styles[attr]);
+}
+
+export function closestAttr(s, t, element) {
+	let el = element;
+
+	do {
+		if (
+			matchesAttr(el, s, t)
+		) {
```

Comment:
We should be fine to have the condition on one line
```suggestion
		if (matchesAttr(el, s, t)) {
```

Reply:
Thanks, updated